setup.py

```diff
@@ -379,9 +379,9 @@ def run(self):
           'https://github.com/morinted/dmgbuild/releases/download/v1.2.1%2Bplover/dmgbuild-1.2.1.pex',
           '548a5c3336fd30b966060b84d86faa9a697b7f94'
         )
-        subprocess.check_call(('chmod', '+x', dmgbuild))
         dmg = 'dist/%s.dmg' % PACKAGE
-        cmd = '%s -s osx/dmg_resources/settings.py Plover %s' % (dmgbuild, dmg)
+        # Use Apple's built-in python2 to run dmgbuild
+        cmd = '/usr/bin/python %s -s osx/dmg_resources/settings.py Plover %s' % (dmgbuild, dmg)
```

Comment:
Both `dmgbuild` and `dmg` are relative paths, and we (indirectly) set the basenames, so it should be okay.